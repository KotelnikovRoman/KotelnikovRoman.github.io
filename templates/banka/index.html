<!DOCTYPE html>
<html lang="ru" class="no-js">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="css/reset.css">
    <link rel="stylesheet" href="css/style.css">
    <title>Пенькоф</title>
</head>
<body>
    <header class="header">
        <div class="container menu_hide">
            <div class="header-wrap">
                <div class="logo">Пенькоф</div>
                <!-- /.logo -->
                <nav class="nav">
                    <a href="" class="nav__link">Банки</a>
                    <a href="" class="nav__link">Чулки</a>
                    <a href="" class="nav__link">Под подушкой</a>
                    <a href="" class="nav__link">Страховка</a>
                    <a href="" class="nav__link">Кэшбэк</a>
                </nav>
                <!-- /.nav -->
                <a href="" class="login">Войти</a>
            </div>
            <!-- /.header-wrap -->
        </div>
        <!-- /.container -->
    </header>
    <section class="section">
        <div class="container">
            <div class="tabs">
                <h1 class="tabs-logo">Penkoff Blank</h1>
                <!-- /.tabs-logo -->
                <ul class="tabs-menu">
                    <li class="tabs-menu__item">Обзор</li>
                    <!-- /.tabs-menu__item -->
                    <li class="tabs-menu__item">Примеры</li>
                    <!-- /.tabs-menu__item -->
                    <li class="tabs-menu__item">Документы</li>
                    <!-- /.tabs-menu__item -->
                    <li class="tabs-menu__item">Как пользоваться</li>
                    <!-- /.tabs-menu__item -->
                </ul>
                <!-- /.tabs-menu -->
            </div>
            <!-- /.tabs -->
            <div class="tabs-content">
                <h2 class="tabs-content__title">Храните деньги в банке <br> это надёжно и безопасно</h2>
                <!-- /.tabs-content__title -->
                <p class="tabs-content__text">Вы можете быть уверены, что ваше деньги в надёжном месте, если они в&nbsp;трёхлитровой банке</p>
                <!-- /.tabs-content__text -->
                <button class="button">Заказать банку</button> <!-- /.button -->
            </div>
            <!-- /.tabs-content -->
        </div>
    </section>
    <section class="features">
        <div class="container">
            <h2 class="section-title section__title">Трёхлитровая банка с  двойным дном</h2>
            <div class="card card-full card_bottom">
                <h3 class="card__title">Часть денег проваливается на дно</h3>
                <p class="card__text">К концу месяца у вас накапливается сумма 10% от&nbsp;всех пополнений. Можно смело тратить их на мечту</p>
            </div>
            <!-- /.card card-full -->
            <div class="card-wrap">
                <div class="card section__card">
                    <h3 class="card__title">Деньги дома</h3>
                    <!-- /.card__title -->
                    <p class="card__text">Когда Деньги лежат дома, у вас просто нет возможности потратить их на всякую ерунду</p>
                    <!-- /.card__text -->
                </div>
                <!-- /.card -->
                <div class="card section__card">
                    <h3 class="card__title">Без комиссии</h3>
                    <!-- /.card__title -->
                    <p class="card__text">Пополните банку без комиссии. Доставать деньги можно также без комисси и штрафов</p>
                    <!-- /.card__text -->
                </div>
                <!-- /.card -->
                <div class="card section__card">
                    <h3 class="card__title">Копейка рубль бережёт</h3>
                    <!-- /.card__title -->
                    <p class="card__text">Вы можете бросать в банку даже мелочь после покупак, которая завалялась в кармане</p>
                    <!-- /.card__text -->
                </div>
                <!-- /.card -->
            </div>
            <!-- /.card-wrap -->
        </div>
        <!-- /.container -->
    </section>
    <!-- /.features -->
    <section class="best">
        <div class="container">
            <h2 class="section-title">Лучшая банка в мире</h2>
            <!-- /.section-title -->
            <span class="section-subtitle">По версии мамы, бабушки и тёти</span>
            <!-- /.section-subtitle -->
        </div>
        <!-- /.container -->
    </section>
    <!-- /.best -->
    <script src="js/modernizr-custom.js"></script>
    <script src="js/jquery-3.3.1.js"></script>
    <script src="js/script.js"></script>
</body>
</html>