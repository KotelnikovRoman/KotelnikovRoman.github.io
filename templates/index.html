<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<meta name="keywords" content="шаблоны сайтов, шаблоны сайтов html, шаблоны сайтов бесплатно, шаблоны сайтов скачать бесплатно, готовые шаблоны сайтов, бесплатные шаблоны сайтов html, шаблоны сайтов joomla, простые шаблоны сайтов html, шаблоны сайтов html5, продажа шаблонов сайтов, готовые шаблоны сайтов бесплатно, шаблоны сайтов на вордпресс, шаблоны веб сайтов">
	<title>Шаблоны сайтов</title>
</head>
<body>
	
</body>
</html>