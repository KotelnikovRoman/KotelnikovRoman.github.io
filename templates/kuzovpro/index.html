<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="description" content="Мы производим детали для кузовного ремонта автомобилей и отправляем их по всей России с 2015 года. Устанавливаем ремонтные детали и восстанавливаем кузов">
    <link rel="stylesheet" href="css/style.css">
    <title>Document</title>
</head>
<body>
    <header class="header">
        <div class="container container__flex">
            <div class="logo">
                <h1 class="logo__title">
                    кузовпро
                </h1>
            </div>
            <!-- /.logo -->
            <nav class="nav">
                <ul class="nav__list">
                    <li class="nav__item">
                        <a href="" class="nav__link">
                            главная
                        </a>
                    </li>
                    <li class="nav__item">
                        <a href="" class="nav__link">
                            примеры установки
                        </a>
                    </li>
                    <li class="nav__item">
                        <a href="" class="nav__link">
                            доставка и оплата
                        </a>
                    </li>
                    <li class="nav__item">
                        <a href="" class="nav__link">
                            контакты
                        </a>
                    </li>
                    <li class="nav__item">
                        <a href="" class="nav__link">
                            каталог деталей
                        </a>
                    </li>
                    <li class="nav__item">
                        <a href="" class="nav__link">
                            отследить заказ
                        </a>
                    </li>
                </ul>
            </nav>
            <!-- /.nav -->
            <div class="phone">
                <p class="phone__number">
                    +7 (964) 732 20 73
                </p>
            </div>
            <!-- /.phone -->
        </div>
        <!-- /.container -->
    </header>
    <main class="main">
        <div class="container"></div>
        <!-- /.container -->
    </main>
    <footer class="footer">
        <div class="container"></div>
        <!-- /.container -->
    </footer>
</body>
</html>