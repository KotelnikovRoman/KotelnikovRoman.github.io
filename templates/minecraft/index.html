<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
</head>

<body>
    <header class="header">
        <div class="container"></div>
        <!-- /.container -->
    </header>
    <!-- /.header -->
    <main class="main">
        <div class="container"></div>
        <!-- /.container -->
    </main>
    <!-- /.main -->
    <footer class="footer">
        <div class="container"></div>
        <!-- /.container -->
    </footer>
    <!-- /.footer -->
</body>

</html>