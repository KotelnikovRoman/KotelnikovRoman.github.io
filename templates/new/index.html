<!DOCTYPE html>
<html lang="ru">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <link href="https://fonts.googleapis.com/css?family=Roboto:400,700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="css/reset.css">
    <link rel="stylesheet" href="css/style.css">
    <title>Пенькоф</title>
</head>
<body>
    <header>
        <div class="container">
            <div class="header-wrap">
                <div class="logo">Пенькоф</div>
                <!-- /.logo -->
                <nav class="nav">
                    <a href="" class="nav__link">Банки</a>
                    <a href="" class="nav__link">Чулки</a>
                    <a href="" class="nav__link">Под подушкой</a>
                    <a href="" class="nav__link">Страховка</a>
                    <a href="" class="nav__link">Кэшбэк</a>
                </nav>
                <!-- /.nav -->
                <a href="" class="login">Войти</a>
            </div>
            <!-- /.header-wrap -->
        </div>
        <!-- /.container -->
    </header>
    <script src="js/jquery-3.3.1.js"></script>
    <script src="js/script.js"></script>
</body>
</html>