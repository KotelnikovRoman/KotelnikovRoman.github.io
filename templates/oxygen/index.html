<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="preconnect" href="https://fonts.gstatic.com">
    <link href="https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@1,300&family=Montserrat:wght@300;700&family=Open+Sans:wght@300;400;600;700&family=Raleway:wght@700&display=swap" rel="stylesheet">
    <link rel="stylesheet" href="css/style.css">
    <title>Oxygen</title>
</head>
<body>
    <nav class="nav">
        <div class="nav__logo">
            
        </div>
        <!-- /.nav__logo -->
        <ul class="nav__menu">
            <li class="nav__item">
                <a href="" class="nav__link">home</a>
            </li>
            <li class="nav__item">
                <a href="" class="nav__link">about us</a>
            </li>
            <li class="nav__item">
                <a href="" class="nav__link">portfolio</a>
            </li>
            <li class="nav__item">
                <a href="" class="nav__link">our team</a>
            </li>
            <li class="nav__item">
                <a href="" class="nav__link">blog</a>
            </li>
            <li class="nav__item">
                <a href="" class="nav__link">contact</a>
            </li>
        </ul>
        <!-- /.nav__menu -->
    </nav>
    <!-- /.nav -->
    <header class="header"></header>
    <main class="main"></main>
    <footer class="footer"></footer>
</body>
</html>