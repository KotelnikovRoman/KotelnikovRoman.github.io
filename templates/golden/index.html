<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="css/style.css">
</head>
<body>
	<header>
		<div class="container">
			<div class="heading clearfix">
				<img src="img/logo.png" alt="Golden" class="logo">
				<nav>
					<ul class="menu">
						<li><a href="">home</a></li>
						<li><a href="">Services</a></li>
						<li><a href="">Portfolio</a></li>
						<li><a href="">About</a></li>
						<li><a href="">Contact</a></li>
					</ul>
				</nav>
			</div><!-- heading -->
			<div class="titles">
				<div class="title__first">
					Welcome To Our Studio!
				</div>
				<h1>It’s nice to Meet you</h1>
			</div><!-- titles -->
			<a href="" class="button">Tell Me More</a>
		</div>
	</header>
	<section id="services">
		<div class="container">
			<div class="title">
				<h2>Services</h2>
				<p>Proin iaculis purus consequat sem cure</p>
			</div>
			<div class="services clearfix">
				<div class="services_item">
					<img src="img/icon3.png" alt="">
					<h3>E-Commerce</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="services_item">
					<img src="img/icon2.png" alt="">
					<h3>Responsive Web</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
				<div class="services_item">
					<img src="img/icon3.png" alt="">
					<h3>Web Security</h3>
					<p>Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit 
					aenean rhoncus posuere odio in tincidunt.</p>
				</div>
			</div>
		</div>
	</section>
	<section id="portfolio">
		<div class="container">
			<div class="title">
				<h2>Our portfolio</h2>
				<p>Proin iaculis purus consequat sem cure</p>
			</div>
			<div class="works clearfix">
				<img src="img/img1.jpg" alt="">
				<img src="img/img2.jpg" alt="">
				<img src="img/img3.jpg" alt="">
			</div>
		</div>
	</section>
	<section id="about">
		<div class="container">
			<div class="title">
				<h2>about us</h2>
				<p>Proin iaculis purus consequat sem cure</p>
			</div>
			<div class="about_article clearfix">
				<div class="left">
					<div class="about_img">
						<img src="img/about1.jpg" alt="">
					</div>
					<div class="about_content">
						<h2>July 2010</h2>
						<h3>Our Humble Beginnings</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
					</div>
				</div>
			</div><!-- end about_article -->
			<div class="about_article clearfix">
				<div class="right">
					<div class="about_img">
						<img src="img/about2.jpg" alt="">
					</div>
					<div class="about_content">
						<h2>January 2011</h2>
						<h3>Facing Startup Battles</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
					</div>
				</div>
			</div><!-- end about_article -->
			<div class="about_article clearfix">
				<div class="left">
					<div class="about_img">
						<img src="img/about3.jpg" alt="">
					</div>
					<div class="about_content">
						<h2>December 2012</h2>
						<h3>Enter The Dark Days</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
					</div>
				</div>
			</div><!-- end about_article -->
			<div class="about_article clearfix">
				<div class="right">
					<div class="about_img">
						<img src="img/about4.jpg" alt="">
					</div>
					<div class="about_content">
						<h2>february 2014</h2>
						<h3>Our Triumph</h3>
						<p>Proin iaculis purus consequat sem cure 
						digni ssim. Donec porttitora entum suscipit 
						aenean rhoncus posuere odio in tincidunt. Proin 
						iaculis purus consequat sem cure digni 
						ssim. Donec porttitora entum suscipit.</p>
					</div>
				</div>
			</div><!-- end about_article -->
		</div><!-- end container -->
	</section>
	<footer>
		<div class="container">
			<p>© Copyright 2014 FreebiesXpress.com</p>
		</div>
	</footer>
</body>
</html>